content/en/docs/tasks/configure-pod-container/configure-pod-configmap.md

```diff
@@ -506,7 +506,7 @@ configmap/special-config-2-c92b5mmcf2 created
 ## Configure all key-value pairs in a ConfigMap as container environment variables
 
 {{< note >}}
-This functionality is available in Kubernetes v1.6 and later.
+This functionality is available in Kubernetes v1.16 and later.
```

Comment:
```suggestion
This functionality is available in Kubernetes v1.16 and later.
```
I'm not sure whether the version should be 1.16 or something else. Let's wait for other reviewers to confirm it.

Reply:
understood.

Reply:
Even if this is available since 1.16, the note can be safely dropped because 1.16 is a long history ago.

Reply:
sounds good! There are a lot of such notes on the docs. Should we prefer removing all the notes that are pointing to very old versions?

Reply:
Yup. Those old information are no longer useful anyway.

Reply:
@tamilselvan1102 Better to remove the line then.

Reply:
> @tamilselvan1102 Better to remove the line then.

I am confused, please share me title you want. i will update